### PR TITLE
[PVR] Trac 17372: Fix wrong initial channel focus with 'Continue last channel on startup' setting.

### PR DESCRIPTION
**Pull request by ksooo**
Fixes Trac 17372: http://trac.kodi.tv/ticket/17372

Fix was runtime tested on macOS on latest Kodi master.

@Jalle19 for review?

### CODE REVIEW

**Comment by ksooo** [comment]
jenkins build this please

**Comment by ksooo** [comment]
IOS Jenkins error is unrelated.

<pre>
make: *** [/Users/Shared/jenkins/workspace/IOS/tools/depends/pre-depends/autoconf-pre-depends/../../pre-build-deps/autoconf-2.68.tar.gz] Error 7
autoconf was not found and could not be built. Aborting.
</pre>

File: xbmc/epg/GUIEPGGridContainer.h [hunk 1]
```diff
@@ -82,8 +82,8 @@ namespace EPG
     void GoToEnd();
     void GoToNow();
     void SetTimelineItems(const std::unique_ptr<CFileItemList> &items, const CDateTime &gridStart, const CDateTime &gridEnd);
-    void SetChannel(const PVR::CPVRChannelPtr &channel);
-    void SetChannel(const std::string &channel);
+    bool SetChannel(const PVR::CPVRChannelPtr &channel);
```
Comment (Jalle19) [inline xbmc/epg/GUIEPGGridContainer.h]: Might want to document what the return value means

**Review by Jalle19 — APPROVED** [review]

**Comment by ksooo** [comment]
@Jalle19 requested changes are in.

jenkins build and merge

**Pull request merged**